src/TWIG/Pages/index.twig

```diff
@@ -0,0 +1,37 @@
+{% extends '../Layouts/layout.twig' %}
+
+{% block title %}
+  index page
+{% endblock %}
+
+{% block content %}
+  {% embed './article.twig' %}
```

Comment:
you should never embed or extend a page template inside another sibling page template, please remove both embeds from this page